src/main/java/org/pac4j/j2e/filter/RequiresAuthenticationFilter.java

```diff
@@ -81,52 +75,44 @@ public void init(final FilterConfig filterConfig) throws ServletException {
             final Config config = ConfigBuilder.build(configFactoryParam);
             ConfigSingleton.setConfig(config);
         }
-
         this.clientName = getStringParam(filterConfig, Pac4jConstants.CLIENT_NAME, this.clientName);
-        CommonHelper.assertNotNull(Pac4jConstants.CLIENT_NAME, this.clientName);
-
-        this.requireAnyRole = getStringParam(filterConfig, Pac4jConstants.REQUIRE_ANY_ROLE, this.requireAnyRole);
-        this.requireAllRoles = getStringParam(filterConfig, Pac4jConstants.REQUIRE_ALL_ROLES, this.requireAllRoles);
         this.authorizerName = getStringParam(filterConfig, Pac4jConstants.AUTHORIZER_NAME, this.authorizerName);
-
-        this.useSessionForDirectClient = getBooleanParam(filterConfig, Pac4jConstants.USE_SESSION_FOR_DIRECT_CLIENT, this.useSessionForDirectClient);
-        this.allowDynamicClientSelection = getBooleanParam(filterConfig, Pac4jConstants.ALLOW_DYNAMIC_CLIENT_SELECTION, this.allowDynamicClientSelection);
```

Comment:
Remove this option. A list of client names in the `clientName` parameter does the job in an easier way.